system/netdata.service.in

```diff
@@ -1,15 +1,56 @@
+# netdata systemd target
+
 [Unit]
-Description=Real time performance monitoring
+Description=netdata - Real-time performance monitoring
+Documentation=man:netdata
+Documentation=file:///usr/share/doc/netdata/html/index.html
+Documentation=https://github.com/firehol/netdata
 After=network.target httpd.service squid.service nfs-server.service mysqld.service named.service postfix.service
+Wants=network-online.target
+ConditionPathExists=/etc/netdata/netdata.conf
 
 [Service]
 Type=simple
+Environment="netdata_LOG_LOCATION=/var/log/netdata/log"
+ExecStart=@sbindir_POST@/netdata -D
+ExecReload=@sbindir_POST@/netdata reload
+TimeoutStopSec=60
+KillMode=mixed
+KillSignal=SIGTERM
+OOMScoreAdjust=-900
+
 User=netdata
 Group=netdata
-ExecStart=@sbindir_POST@/netdata -D
+Restart=on-abnormal
+RestartSec=2s
+LimitNOFILE=65536
 
-# saving a big db on slow disks may need some time
-TimeoutStopSec=60
+WorkingDirectory=/tmp
+
+# Hardening
+
+NoNewPrivileges=false
+PermissionsStartOnly=true
+# CAP_SETGID is required for setgroups()
+CapabilityBoundingSet=CAP_DAC_READ_SEARCH CAP_SYS_PTRACE CAP_SETGID
+PrivateTmp=true
+ProtectHome=read-only
+ProtectSystem=full
+#SystemCallFilter=~@cpu-emulation @debug @keyring @module @mount @obsolete @raw-io @resources
+
+ReadOnlyDirectories=/
+ReadWriteDirectories=/proc/self
+ReadWriteDirectories=/var/lib/netdata
+ReadWriteDirectories=/var/log/netdata
+ReadWriteDirectories=/var/cache/netdata
+
+# Access to devices and kernel modules and tunables is required
+PrivateDevices=no
+ProtectKernelModules=no
+ProtectKernelTunables=no
+
+StandardOutput=syslog+console
+StandardError=syslog+console
```

Comment:
"syslog+console" is meant to capture any output before the logging subsystem is initialized (or if it cannot write to disk for any reason...) as you wrote at it will still log on console while also logging on syslog.